styles/styles.css

```diff
@@ -54,11 +54,15 @@ html:has(dialog[open]) {
 body {
   margin: 0;
   overflow-x: hidden;
-  display: flex;
+  display: none;
```

Comment:
Do you remember why you initially changed it to flex here https://github.com/headwire-edge-delivery/eds-self-service-website/commit/c9a69ae43d185e61ab1975d42247c11035bf2ea8#diff-37f6471b306924bbb531d040e7ab55df5f83511246a411d22947462323516afaR35 ?

Reply:
It looks like here I changed the default block to flex. I made it flex so the footer could be moved to the bottom when there isn't enough content.

I think the AEM default of `body {display: none;} body.appear {display: block;}` must have been removed a while back